lighthouse-plugin-publisher-ads/audits/idle-network-times.js

```diff
@@ -195,7 +195,7 @@ function checkIfTagIsBlocking(idlePeriod, blockingTags, pageStartTime) {
 /**
  * Checks waiting on load events is the cause of the idle period.
  * @param {IdlePeriod} idlePeriod
- * @param {LH.Artifacts.TraceTimes} timings
+ * @param {LH.Artifacts.NavigationTraceTimes} timings
```

Comment:
this was for internal trace processing work in https://github.com/GoogleChrome/lighthouse/pull/12633. We didn't realize anyone was using this type directly. Name change is mostly to clarify in future non-navigation LH runs the expected timing fields won't be available, and you need a `NavigationTraceTimes` for that.